doc/README.modpack_installer

```diff
@@ -209,64 +188,60 @@ here, and followed by one line for each modpack:
  - "license":
      Free text summarising the distribution terms for the modpack
      content (by naming a well-known license, not quoting the full
-     license text!)
+     license text!). Consider using SPDX identifiers
+     (https://spdx.org/licenses/).
  - "URL":
-     The URL to a .mpdl file for the individual modpack.
-     Either a relative URL starting "./" (in which case it's relative to
-     the URL of modpack.list) or an absolute URL (which can be on some
-     other web server).
+     The URL to a .json file for the individual modpack.
+     The URL can be either relative (in which case it's relative to
+     the URL of modpack.list) or absolute (which can be on some other
+     web server).
  - "notes":
      Optional free text; usually shown as a tooltip.
 
 
-2. *.mpdl control file, defining an individual modpack
+2. *.json control file, defining an individual modpack
 
 This is the core control file for a modpack, specifying what files it
 contains, where to download them from, and where they are installed.
 
-Most modpack authors will just publish the URL of an .mpdl file
+Some modpack authors will just publish the URL of an .json file
 directly, for users to give to the modpack installer tool. (There
 doesn't have to be a modpack.list file anywhere that refers to the
-.mpdl file.)
-
-Again, this is a file in Freeciv's "spec-file" format, the same as is
-used for modpacks like rulespecs, tilespecs, etc; see those for examples
-of the general syntax. Its filename must end in ".mpdl".
-
-Here is an example of a .mpdl file:
-
-[info]
-options = "+Freeciv-mpdl-Devel-3.1-2019.Jul.15"
-baseURL = "./ancients-6f88ee23de"
-name    = "Ancients"
-type    = "Modpack"
-version = "3.1-0"
-
-[dependencies]
-list =
-{ "modpack", "URL", "type", "version"
-  "Ancients-tiles", "./ancients-tiles.mpdl", "Tileset", "3.1-0"
+.json file.)
+
+Again, this is a file in standard JSON format. Its filename must end
+in ".json".
+
+Here is an example of a modpack control file:
+{
+  "info": {
```

Comment:
Should modpacks be fc21 version specific? They are "ish" with classic fc. There is a control code in the header that is read and parsed.

Reply:
Good question. I would like to keep backward compatibility as much as possible, but sometimes it's just too much work. I also kind of have a plan to get rid of spec files altogether (maybe using Lua, YAML or some other outsourced parser), which will probably induce some compatibility breach… can keep both versions available in the transition period.

I think the modpack index will be version specific using one branch per major version (see #350). It will require some maintenance but it's not so bad that the maintainers are aware of how much work it takes to port a modpack.